perf/flame.jl

```diff
@@ -1,6 +1,8 @@
 # flame.jl: provides allocation breakdown for individual backtraces for single-process unthredded runs
 # and check for overall allocation limits based on previous runs
 # copied and modified from `ClimaAtmos/perf`
+#
+# To run this, add ProfileCanvas to the ClimaEarth environment
```

Comment:
This should be added to flame_diff.jl and flame_test.jl too

Reply:
They don't seem to need ProfleCanvas

Reply:
Because they import ProfileCanvasDiff (which copies the code of ProfileCanvas)